internal/commands/scan.go

```diff
@@ -2228,19 +2238,31 @@ func validateCreateScanFlags(cmd *cobra.Command) error {
 	}
 	exploitablePath, _ := cmd.Flags().GetString(commonParams.ExploitablePathFlag)
 	lastSastScanTime, _ := cmd.Flags().GetString(commonParams.LastSastScanTime)
+	exploitablePath = strings.ToLower(exploitablePath)
 	if !strings.Contains(strings.ToLower(actualScanTypes), commonParams.SastType) &&
 		(exploitablePath != "" || lastSastScanTime != "") {
 		return errors.Errorf("Please to use either --exploitable-path or --last-sast-scan-time flags in SCA, " +
 			"you must enable SAST scan type.")
 	}
-	if !strings.EqualFold(exploitablePath, "true") && !strings.EqualFold(exploitablePath, "false") && exploitablePath != "" {
-		return errors.Errorf("Invalid value for --exploitable-path flag. The value must be true or false.")
+	if exploitablePath != "" {
+		_, err := strconv.ParseBool(exploitablePath)
```

Comment:
The parseBool accepts a lot of values, we should only accept true or false. 

ParseBool returns the boolean value represented by the string. It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False. Any other value returns an error.